h5p-editor-bildetema-words-topic-image/package.json

```
@@ -27,7 +22,8 @@
     "storybook-custom": "*",
     "ts-node": "^10.9.1",
     "tsconfig-custom": "*",
-    "vite-utils": "*"
+    "vite-utils": "*",
+    "vitest": "*"
```

```suggestion
    "vitest": "*"
```
```suggestion
    "vitest": "^2.0.5"
```

This doesn't really matter, but let's keep it the same as everywhere else.